docs/src/main/asciidoc/deploying-to-openshift.adoc

```diff
@@ -56,8 +57,15 @@ By adding this dependency, we now have the ability to configure the OpenShift re
 The configuration items that are available can be found in: `io.quarkus.kubernetes.deployment.OpenShiftConfig` class.
 Furthermore, the items provided by `io.quarkus.deployment.ApplicationConfig` affect the OpenShift resources.
 
-=== Building
+=== Build and deply (in separate steps)
 
+If OpenShift extension was not included during the bootstraping of the project nor added subsequently  (check pom.xml file for it), then add OpenShift as a dependency before building:
```

Comment:
```suggestion
If the OpenShift extension was not included during the bootstraping of the project nor was it added subsequently  (check pom.xml file for it), then it can be added like so:
```

Reply:
Thanks. Done.